frontend/src/partners/PartnerForm.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react'
+import { Formik, Form, Field } from 'formik'
+import { Link } from 'react-router-dom'
+import Input from '../Input'
+import Button from '../Button'
+import PopUp from '../PopUp'
+import schema from './ValidationPartner'
+import RadioInputGroup from '../RadioInputGroup'
+import RadioInput from '../RadioInput'
+
+function PartnerForm({
+  sent,
+  handleClick,
+  popUpMessage,
+  sendData,
+  title,
+  partner,
+}) {
+  return (
+    <div className="container my-5">
+      <div className="row justify-content-center">
+        <div className="col-12">
+          {sent && <PopUp handleClick={handleClick} body={popUpMessage} />}
+          <Formik
+            initialValues={{
+              partnerEmail: '',
+              telefon: '',
+              megrendeloTipusa: '',
+              nev: '',
+              rovidNev: '',
+              adoszam: '',
+              kadoszamtipus: '',
+              bankszamlaszam: '',
+              szamlazasiCimOrszagKod: '',
+              szamlazasiCimOrszagNev: '',
+              szamlazasiCimMegyeNev: '',
+              szamlazasiCimIranyitoszam: '',
+              szamlazasiCimTelepulesNev: '',
+              szamlazasiCimKerulet: '',
+              szamlazasiCimKozteruletNev: '',
+              szamlazasiCimKozteruletJellegNev: '',
+              szamlazasiCimHazszam: '',
+              szamlazasiCimEpulet: '',
+              szamlazasiCimLepcsohaz: '',
+              szamlazasiCimSzint: '',
+              szamlazasiCimAjto: '',
+              szamlazasiCimHrsz: '',
+              ...partner,
+            }}
+            validationSchema={schema}
+            onSubmit={(values) => {
+              sendData(values)
+            }}
+          >
+            {({ values, errors, touched }) => {
+              return (
+                <Form>
+                  <h1 className="text-center">{title}</h1>
+                  <Input name="partnerEmail" label="Email cím" type="text" />
+                  <Input name="telefon" label="Telefonszám" type="text" />
+                  <RadioInputGroup
+                    className="inputStyle"
+                    id="megrendeloTipusa"
+                    label="Megrendelő típusa:"
+                    value={values.megrendeloTipusa}
+                    error={errors.megrendeloTipusa}
+                    touched={touched.megrendeloTipusa}
+                  >
+                    <Field
```

Comment:
Lehet a választható lehetőségeket egy options-ben kellene átadni a RadioInputGroupnak, ami az options alapján pakolná be a RadioInputokat.

Reply:
És a RadioInputGroup-on belül lehetne useField-ezni, így nem kellene neki odapasszolgatni a value-t, error-t, touched-ot, csak a name-et.